src/api/getTimezoneOptions.ts

```diff
@@ -0,0 +1,15 @@
+import {API_ENDPOINT} from '@env';
+import axios from 'axios';
+
+export default async function getTimezoneOptions(userTz: string): Promise<{
+  success: boolean;
```

Comment:
i think we can exctract the return type?

Reply:
Done